### ISSUE TITLE
console text doesn't support shift-jis?

### ISSUE THREAD

**Issue by gesori-pro** [Solybum/PSO-Tools#1]
![image](https://user-images.githubusercontent.com/29522474/117561860-68573380-b0d5-11eb-9014-6d66c5e4762e.png)

hi, I'm trying to translate PSO EP 1&2 GC version
and I tried to make pr2, pr3 to JSON and as you can see it's showing as weird
is there way to can convert to Shift-jis?

**Comment by Solybum** [comment]
Yeah, the library used to read the text (my own) doesn't have a method to read S-JIS, could you either post or send me these files to test?

I suppose the data itself is fine (null terminated) so there shouldn't be any issue reading or writing the text, but I'd have to add a new flag so it supports S-JIS (and possibly any other language not UCS-2

**Comment by gesori-pro** [comment]
[Episode 1&2.zip](https://github.com/Solybum/PSO-Tools/files/6447087/Episode.1.2.zip)
[Episode 3.zip](https://github.com/Solybum/PSO-Tools/files/6447088/Episode.3.zip)

here it is :)

**Comment by Solybum** [comment]
Alright, I'll try to look into this in the next few days, I suppose it might not be that hard to add support for it, but I am a bit busy with work.

**Comment by gesori-pro** [comment]
thank you so much
